register.php

```diff
@@ -15,29 +15,44 @@
 <body>
 <form action="/src/validation/registrationValidation.php" method="post">
 
-    <div class="container">
-        <p class="text-warning-emphasis w-50">If you are not part of the Blue Atomic team, this does not currently benefit you. We may implement features for registered accounts later.</p>
-
-        <label for="username"><b>Username</b></label>
-        <input type="text" placeholder='Your preferred username' name="username" id="username" required>
-
-        <label for="mail"><b>Mail address</b></label>
-        <input type="email" placeholder="example@gmail.com" name="mail" id="mail" required>
-
-        <label for="psw"><b>Password</b></label>
-        <input type="password" placeholder="Enter Password" name="psw" id="psw" required>
-        <?php
-        if (isset($_SESSION['exCode'])) {
-            echo '<p class="text-warning">' . $_SESSION['exCode'] . '<br>';
-        }
-        session_destroy()
-        ?>
```

Comment:
You should keep the php script here. You seem to have removed it entirely.

Reply:
fixed in [0b525cd](https://github.com/BlueAtomic/Aternos-guides/pull/3/commits/0b525cd67d96886b405c3e6f603e00de100a7b48)